### ISSUE TITLE
Notification spam on pokemon despawn

### ISSUE THREAD

**Issue by Slivo-fr** [PokemonGoMap/PokemonGo-Map#148]
The map often throw multiple notifications when pokemon in the notification list disapear.

## Expected Behavior
Notification only trigger when a pokemon appear.

## Current Behavior
notifications are also triggered when pokemon disapear

## Steps to Reproduce (for bugs)

1.Add some common pokemon in the notification list
2.Activate notification with sound
3.Wait for pokemon despawn

## History
This issue was already there on the last repository, there was a fix in PR that was never merged : https://github.com/AHAAAAAAA/PokemonGo-Map/pull/2131

**Comment by JZ6** [comment]
still waiting on a pr rebase, if you were following closely, perhaps you could do the work of writing the rebase.

**Comment by Slivo-fr** [comment]
I considered this option, but I did not really follow this closely, and I cannot get enough time to do it myself unfortunately.
Maybe re-dev what was done instead of rebasing if it's too complex, I'm not so used to git rebase :/

Hope bringing this up here can help

**Comment by invisiblek** [comment]
Thing is, I'm not really on board with https://github.com/AHAAAAAAA/PokemonGo-Map/pull/2131 and what it did. Fixing this bug was a happy side effect of it, but the UX that was added didn't feel right imo.

I'll look at this next, maybe hack something in.

**Comment by invisiblek** [comment]
After examining the server response, I believe the way lured pokestops are flagged has changed. Also it doesn't appear the pokestop has any info about the lured pokemon anymore. See notes here: https://github.com/PokemonGoMap/PokemonGo-Map/pull/493

**Comment by invisiblek** [comment]
If I'm right, this issue will no longer be an issue. My theory here is that lured pokemon now act the same as regular ones. If that's the case, we should entertain the idea of dropping a sort of 'dedicated' scanner (or force the searcher to scan more often) at lured pokestops.

**Comment by invisiblek** [comment]
I was wrong... 😝 

Need to come up with a way to handle lured pokemon. It seems the scanner's location needs to be really close to the pokestop or you won't get the info for the lured pokemon (unlike how it used to work before). I have ideas, but implementing them isn't exactly trivial...

**Comment by hhu94** [comment]
@invisiblek so are lured pokemon returned as normal pokemon now? Or are they still in the pokestop info?

**Comment by invisiblek** [comment]
They're still in lure_info, but you don't just get them for 'free' anymore. You have to make a GET_MAP_DETAILS call with a position right next to that pokestop in order to see that info.

Check this out: https://gist.github.com/invisiblek/bfdba78b015acac974753093a238d224

Notice they both have 9QM= for their active_fort_modifier, which means they're lured, but one gives me lure_info, the other doesn't. The api call I made here had coordinates that were right on top of the first one. The other pokestop was a few blocks away.

**Comment by BaIthamel** [comment]
Does this still exist? No new reports for a while.

**Comment by Slivo-fr** [comment]
Seems fixed, didn't had the issue for a while too!

**Comment by BaIthamel** [comment]
Okay i'll close it, if it shows up again new issue. Can only assume it got fixed :)
